apps/site/src/components/Sticker/BaseSticker.module.scss

```diff
@@ -1,3 +1,16 @@
-.stickerContainer {
+.stickerPositionContainer {
+	position: relative;
+	display: flex;
+}
+
+.stickerParent {
+	position: "relative";
```

Comment:
Should this be a position: `relative` be a string?

Reply:
nope